netdata-installer.sh

```diff
@@ -1241,19 +1255,26 @@ get_compatible_kernel_for_ebpf() {
 
   kpkg=
 
-  if [ "${kver}" -ge 005000000 ]; then
+  # Kernel variables
+  rhver8="0080000000"
+  rhver7="0070061810"
```

Comment:
I'd prefer if those 2 variables were called:
```
  rhver8_0="0080000000"
  rhver7_6="0070061810" 
```

Reply:
This code will be removed @mfundul , it is not longer necessary with the new packages. I am only waiting for the final definitions on `kernel-collector` to update the code.